engine/dlib/src/dmsdk/dlib/android.h

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 The Defold Foundation
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+//
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#ifndef DMSDK_ANDROID_H
+#define DMSDK_ANDROID_H
+
+#if defined(ANDROID)
+
+#include <jni.h>
+#include <android_native_app_glue.h> // For ANativeActivity
+
+/*# SDK Android API documentation
+ * Android utility functions
+ *
+ * @document
+ * @name Android
+ * @namespace dmAndroid
+ * @path engine/dlib/src/dmsdk/dlib/android.h
+ */
+
+namespace dmAndroid {
```

Comment:
Added a new dmsdk header, and a new namespace.

Reply:
I love it!!! 🥰